src/components/interface/MellowLpDepositForm/components/LPMellowVaultDepositWindow.tsx

```diff
@@ -40,7 +40,13 @@ const LPMellowVaultDepositWindow: React.FunctionComponent<LPMellowVaultDepositWi
 
   return (
     <FormPanel
-      sx={{ width: '100%', maxWidth: '328px', background: '#19152A', padding: '16px 16px' }}
+      sx={{
+        width: '100%',
+        maxWidth: '328px',
+        maxHeight: '444px',
+        background: '#19152A',
+        padding: '16px',
```

Comment:
```suggestion
        padding: (theme) => theme.spacing(4)
```

Reply:
fixed in a880d81b18fc358da6db21e427119ac7f03e85ef